orchestration/templates/ingest-clinvar-archive.yaml

```diff
@@ -75,8 +75,28 @@ spec:
                   value: '{{ .Values.volumes.extractSize }}'
                 - name: storage-class
                   value: '{{ .Values.volumes.storageClass }}'
+          # 4a) Ingest raw XML blob from GCS to Jade Data Repo.
+          - name: ingest-raw-xml
+            dependencies: [upload-raw-archive]
+            templateRef:
+              name: ingest-file
+              template: main
+            arguments:
+              parameters:
+                - name: api-url
+                  value: '{{ .Values.repo.apiUrl }}'
+                - name: dataset-id
+                  value: '{{ .Values.repo.datasetId }}'
+                - name: profile-id
+                  value: '{{ .Values.repo.profileId }}'
+                - name: gcs-bucket
+                  value: '{{ .Values.gcs.bucketName }}'
+                - name: gcs-file-path
+                  value: '{{ "{{inputs.parameters.gcs-prefix}}" }}/raw/$localXmlName'
+                - name: target-path
+                  value: '{{ .Values.repo.targetPath }}'
```

Comment:
The target path will need to change from run to run, so I don't think it can be a Helm var.

I've been trying to think about the right value to use. We could use the execution timestamp to ensure we don't try to overwrite existing files, but that doesn't really connect to anything in the ClinVar data. Alt: we could use the release date, which would line up nicely with the data but then we'd need a way to detect if/when the file already exists (we might need that anyway if we ever need to re-submit a partially-failed workflow). WDYT?

Reply:
cool Jade has an endpoint for lookup by path https://jade.datarepo-dev.broadinstitute.org/swagger-ui.html#/repository/lookupFileByPath , I'll add this in where needed